metrics_utility/test/gather/test_jobhostsummary_gather.py

```diff
@@ -0,0 +1,105 @@
+file_paths = f'./metrics_utility/test/test_data/data/2025/06/13/{uuid}-*.tar.gz'
+
+
+@pytest.fixture
+def cleanup_glob():
+    yield
+    for file in glob.glob(file_glob):
+        os.remove(file)
+
+
+test_lines = [
+    'id,created,modified,host_name,host_remote_id,ansible_host_variable,'
+    'ansible_connection_variable,changed,dark,failures,ok,processed,skipped,'
+    'failed,ignored,rescued,job_created,job_remote_id,job_template_remote_id,'
+    'job_template_name,inventory_remote_id,inventory_name,organization_remote_id,'
+    'organization_name,project_remote_id,project_name',
+    '1,2025-06-13 10:00:00+00,2025-06-13 10:00:00+00,default_host_1_2025-06-13,1,'
+    'default_ansible_host,default_ansible_connection,0,0,0,0,0,0,f,0,0,'
+    '2025-06-17 12:33:19.548309+00,1,1,default_unified_job_2025-06-13,1,'
+    'default_inventory_2025-06-13,1,default_org_2025-06-13,1,'
+    'default_unified_job_template_2025-06-13',
+    '2,2025-06-13 10:00:00+00,2025-06-13 10:00:00+00,default_host_2_2025-06-13,2,'
+    'default_ansible_host,default_ansible_connection,0,0,0,0,0,0,f,0,0,'
+    '2025-06-17 12:33:19.548309+00,1,1,default_unified_job_2025-06-13,1,'
+    'default_inventory_2025-06-13,1,default_org_2025-06-13,1,'
+    'default_unified_job_template_2025-06-13',
+    '3,2025-06-13 10:00:00+00,2025-06-13 10:00:00+00,default_host_1_2025-06-13,1,'
+    'default_ansible_host,default_ansible_connection,0,0,0,0,0,0,f,0,0,'
+    '2025-06-17 12:33:19.548309+00,2,1,default_unified_job_2025-06-13,1,'
+    'default_inventory_2025-06-13,1,default_org_2025-06-13,1,'
+    'default_unified_job_template_2025-06-13',
+    '4,2025-06-13 10:00:00+00,2025-06-13 10:00:00+00,default_host_2_2025-06-13,2,'
+    'default_ansible_host,default_ansible_connection,0,0,0,0,0,0,f,0,0,'
+    '2025-06-17 12:33:19.548309+00,2,1,default_unified_job_2025-06-13,1,'
+    'default_inventory_2025-06-13,1,default_org_2025-06-13,1,'
+    'default_unified_job_template_2025-06-13',
+    '5,2025-06-13 10:00:00+00,2025-06-13 10:00:00+00,default_host_1_2025-06-13,1,'
+    'default_ansible_host,default_ansible_connection,0,0,0,0,0,0,f,0,0,'
+    '2025-06-17 12:33:19.548309+00,3,1,default_unified_job_2025-06-13,1,'
+    'default_inventory_2025-06-13,1,default_org_2025-06-13,1,'
+    'default_unified_job_template_2025-06-13',
+    '6,2025-06-13 10:00:00+00,2025-06-13 10:00:00+00,default_host_2_2025-06-13,2,'
+    'default_ansible_host,default_ansible_connection,0,0,0,0,0,0,f,0,0,'
+    '2025-06-17 12:33:19.548309+00,3,1,default_unified_job_2025-06-13,1,'
+    'default_inventory_2025-06-13,1,default_org_2025-06-13,1,'
+    'default_unified_job_template_2025-06-13',
+]
+
+
+@pytest.mark.filterwarnings('ignore::ResourceWarning')
+def test_command(cleanup_glob):
+    """Build xlsx report using build command and test its contents."""
+    # Enable it by the second PR, first this PR has to be merged in order to
+    # pytest.yml works, because it is pull request target
+    return
+    """
```

Comment:
can you just @pytest.skip instead?

Or move the test to a separate PR?

There's not much point adding a test that's disabled.